etc/adminhtml/system/adyen_security.xml

```diff
@@ -38,5 +38,17 @@
                 ]]>
             </comment>
         </field>
+        <field id="notification_hmac_key" translate="label" type="text" sortOrder="20" showInDefault="1"
```

Comment:
Please add test and live HMAC key configuration, I just checked the prestashop repo where there is only one but it is much harder to manage :(
When retrieving the configuration check the plugin configuration if test or live mode is enabled and retrieve the hmac key based on that.

Reply:
Also please declare this as a sensitive config - you can find an example in this PR: https://github.com/Adyen/adyen-magento2/pull/707 if help needed please ask @acampos1916

Reply:
done 👍